changelog.txt

```diff
@@ -1 +1,7 @@
-
+feature - Implements anonymous user upgrade functionality with FirebaseUI
```

Comment:
Let's sort as follows:
features first
fixes next

Let's also add:
fixed - Updates closure open source builder to latest version fixing dependency on `eval` and older `marked` module which had some vulnerability issues.